packages/core/src/Controller.ts

```diff
@@ -242,6 +242,28 @@ export class Controller<State extends Indexable = UnknownProps>
     return this
   }
 
+  /** Freeze the active animation in time */
+  pause(keys?: OneOrMore<string>) {
+    if (is.und(keys)) {
+      this.each(spring => spring.pause())
+    } else {
+      const springs = this.springs as Indexable<SpringValue>
+      each(toArray(keys), key => springs[key].pause())
+    }
+    return this
+  }
```

Comment:
I intentionally copy and paste same logic from `stop` method since the logic may be different in the future.